Improve compatibility with Python < 3.9

ddn0 opened the pull request:
While generating documentation with older Python versions, I noticed an
import error with importlib.metadata. I did not check for complete
Python compatibility < 3.9 but this was sufficient for importing the
katana library.

roshandathathri commented on libcusp/include/galois/graphs/TrianglePartitioner.h:
```diff
@@ -18,7 +18,7 @@
- * @file MiningPartitioner.h
+ * @file TrianglePartitioner.h
  *
  * Graph mining partitioning that duplicates edges. Currently only supports an
```
The first sentence can be replaced with "Partitioning that creates edge proxies if an edge exists between any two vertex proxies on the same host. Master proxies for edges and master/mirror proxies for vertices are created similar to that in CuSP."

ddn0 committed:
Improve compatibility with Python < 3.9

While generating documentation with older Python versions, I noticed an
import error with importlib.metadata. I did not check for complete
Python compatibility < 3.9 but this was sufficient for importing the
katana library.

arthurp reviewed (approved):
Sigh. I just thought about this. The problem is that there may be a need to explicitly install `importlib_metadata` it's not in the standard library.
